tornettools/plot.py

```diff
@@ -247,6 +252,23 @@ def __plot_client_goodput(args, torperf_dbs, tornet_dbs):
         yscale="taillog",
         xlabel="Client Transfer Goodput (Mbit/s): 0.5 to 1 MiB")
 
+def __plot_client_goodput_5MiB(args, torperf_dbs, tornet_dbs):
+    # Computes throughput for last of 5MiB transfer
+
+    # cache the corresponding data in the 'data' keyword for __plot_cdf_figure
+    for tornet_db in tornet_dbs:
+        tornet_db['data'] = tornet_db['dataset']
+    for torperf_db in torperf_dbs:
+        # convert tor's microseconds into seconds
+        client_gput = torperf_db['dataset']["client_goodput_5MiB"]
```

Comment:
I'm confused about this.... again I don't think we want to change the units of any of the parsed outputs so that we maintain compatibility with older results.

Also is the old comment in `__plot_client_goodput` wrong? I think it's converting bps to Mbps...?

Reply:
Yes I agree with what you said about old data that would have to be re-parsed. The rationale behind changing this is that it's converting this for the tor network data and not for the tgen sim data which make things IMO a bit confusing.